specification/marketplacecatalog/resource-manager/Products/preview/2022-09-01-preview/products.json

```diff
@@ -0,0 +1,1411 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Catalog Browse API",
+    "description": "Microsoft Marketplace Catalog Browse API",
+    "version": "2022-09-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Marketplace/products/{id}": {
+      "get": {
+        "tags": [
+          "SinglePrivateProduct"
+        ],
+        "summary": "Returns a single product directly from BigCat, by subscription as audience",
+        "operationId": "SinglePrivateProduct_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
```

Comment:
>SubscriptionIdParameter [](http://example.com/codeflow?start=34&length=23)

Instead of defining SubscriptionIdParameter locally, refer it from [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v3/types.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v3/types.json).

Reply:
Fixed